frontend-react/src/content/resources/api-programmers-guide/getting-started/GettingStarted.mdx

```diff
@@ -0,0 +1,114 @@
+* **Method 2:** Shared secret API key
+
+The examples below use the fake client-id healthy-labs that you will change for your submissions. 
+The examples submit the payload contained in the file `./healthy-labs-nonPII-data.csv` (or .hl7). 
+In the examples, data are submitted via an HTTP POST to the ReportStream staging system reports endpoint. 
+The data submitted are sent as the payload of the POST, as is, with no changes.
+
+<Accordion items={[
+    {
+        headingLevel: "h3",
+        title: "Method 1: Token-based authentication with public/private key pair (Recommended)", 
+        content: <Method1 />,
+        id: 1
+    },
+    {
+        headingLevel: "h3",
+        title: "Method 2: Shared secret key authorization", 
+        content: <Method2 />,
+        id: 2
+    },
+]} />
+
+### Test your automation
+
+Once authentication is complete, you can test your automation code as well as your code that handles responses using the staging API. 
+Data is sent in the HTTP payload, either in CSV or HL7 2.5.1 format. 
+You can use curl commands, Postman, or another method of your choosing to post test submissions to the staging environment.
+
+<Alert type="info" headingLevel="div" slim={true} className="border-0 font-sans-xs">**Note:** Do not send any PII or PHI to the staging system — only fake (dummy, example, synthetic) data is acceptable.</Alert>
+
+Let us know when you send submissions to the staging environment. We’ll review that data and work with you to correct any issues. You may send as many fake data submissions to staging as is helpful.
+
+For troubleshooting on your own, here is the complete endpoint input and response [OpenAPI specification](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/api).
+
+<h2 className="rs-numbered">Test real data in production</h2>
+   The ReportStream team will onboard you to the production system in training mode. If using a shared secret key, you’ll receive API keys or tokens and the URL via Keybase. ReportStream won’t forward or transport data received in training mode. However, the response message provides detailed information on where your data would have flowed if production mode was active.
+
+<h2 className="rs-numbered">Start sending data in production</h2>
+   When you are ready, the ReportStream team will move you out of training mode and enable full production mode. Once in production, you can send a single record or up to 10,000 records in a single submission.
+
+Data will automatically flow to appropriate state, local, and federal jurisdictional systems.
+
+{
+<Alert type="info" headingLevel="div" slim={true} className="border-0 font-sans-xs">
+    <span className="text-bold">Note:</span> Some jurisdictions require additional validation before sending data to their systems. If this affects your data submission, the ReportStream team will assist you in the process. Currently, the following states require additional validation:
+    <ul>
```

Comment:
Could have the following instead of html:
* California
* Illinois
* New Jersey
* Washington

Reply:
Due to needing to wrap the alert in brackets I cannot. Everything in the brackets is jsx-only.